ledger-core/vanilla/cachekit/cache_example.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package cachekit
+
+type Key = uint64
+type Value = string
+
+const zeroValue = ""
+
+/*
+	UintCache is an example/template implementation of a cache that uses the Core.
+
+	To create a new cache type - copy this file, rename struct, set Key and Value type aliases accordingly, then
+	set zeroValue constant in accordance with Value type.
+
+
+ */
+
+func NewUintCache(cs Strategy) *UintCache {
+	sc := &UintCache{
```

Comment:
`cs/sc` naming is very bad - adds work to reader without any reason. Use `cache`, `res`, or `result` instead of `sc`. `strat` or `strategy` instead of `cs`.

Reply:
imho, when variable exists only for <10 lines then its name should not be misleading, but can have no meaning.

Reply:
fixed

Reply:
I agree in general, in this case I had to decipher `cs/sc` a few times and it was painful...